### DISCUSSION TITLE
PyMISP upload multiple events in JSON

### DISCUSSION THREAD

**Discussion by swebster96** [MISP/PyMISP#842] [Q&A] [answered]
In terms of uploading multiple Events from a JSON file, is there existing code example or python scripts to test and play with in PyMISP?

**Comment by Rafiot** [comment, marked as answer]
You cannot add more than one event at the time using PyMISP. 

You can find some sample code here: https://github.com/MISP/PyMISP/blob/main/docs/tutorial/FullOverview.ipynb
